src/main/java/SimpleAPI.java

```diff
@@ -334,7 +358,7 @@ else if (file.list().length > 0)
 					if (file.isDirectory()) {
 						if (file.list().length > 0)
 							throw new IllegalArgumentException("Directory is not empty: " + file);
-						resultLocations.put(port, URI.create(file.toURI() + "/"));
+						resultLocations.put(port, file.toURI());
```

Comment:
Why this change?

Reply:
If i recall, It could lead to invalid/malformed path on windows ending by "//" or "\/"

Reply:
I see. Shouldn't we instead change the

```java
if (result.endsWith("/")) {
```
above to
```java
if (result.endsWith("/") || result.endsWith("\\")) {
```
?

Reply:
Indeed, I did not treat this in the results side (only the parameters initialization side was creating issues during developement of the connection between word addin and pipeline)